frontend/src/pages/post/SearchPage.jsx

```diff
@@ -0,0 +1,25 @@
+import { useParams } from "react-router-dom"
+import PostPageView from "./PostPageView"
+import postData from './mock-data'
+
+const SearchPage = () => {
+    const [searchTerm, setSeachTerm] = useState('')
+
+    return (
+        <div className="Search">
+            <input type="text" placeholder="Seach..." onChange={event => {setSeachTerm(event.target.value);}} />
```

Comment:
use the MUI text input component, they probably have search field templates

Reply:
They do